src/renderer/opengl/windows/windows_gl_context.cpp

```diff
@@ -77,7 +77,7 @@ GLContext::GLContext() : m_impl{std::make_unique<Impl>()} {
     int32_t majorVersion{}, minorVersion{};
     glGetIntegerv(GL_MAJOR_VERSION, &majorVersion);
     glGetIntegerv(GL_MINOR_VERSION, &minorVersion);
-    NOX_ASSERT_MSG((majorVersion != glMajorVersion) && (minorVersion != glMinorVersion), "This system doesn't support OpenGL 4.6");
+    NOX_ASSERT_MSG((majorVersion != glMajorVersion) || (minorVersion != glMinorVersion), "This system doesn't support OpenGL 4.6");
```

Comment:
the logic expression was invalid, any incompatibility between either major or minor versions should trigger assertion

Reply:
yep, good catch